### DISCUSSION TITLE
Feature matrix showing which VSTs work under which OSes

### DISCUSSION THREAD

**Discussion by turian** [DBraun/DawDreamer#121] [Ideas]
It would be great to have a feature matrix that shows which VSTs work under which OSes using checkmark, question mark, or X. (Inspired by #120)

Perhaps the simplest way to do it would be a one-click script where you pass it the vst location, it generates random parameters for everything, renders a sound, and sees that the sound isn't mainly empty.

It could then dump the output in a verbose way and give instructions on how to post a PR or mail the dev. That way, we could easily see which VSTs work where.

**Comment by DBraun** [comment]
Yes, I'm very open to contributions like this if I don't get around to it myself. There is https://github.com/DBraun/DawDreamer/wiki/Plugin-Compatibility

**Comment by turian** [comment]
Can you propose a simple script that shows how to test the functionality on something known to work? 

I could try to write a Docker for it then.

**Comment by DBraun** [comment]
I suggest adapting https://github.com/DBraun/DawDreamer/blob/bf874f5a722139015342871af11504a0a9ac4062/tests/test_plugins.py#L42 or https://github.com/DBraun/DawDreamer/blob/bf874f5a722139015342871af11504a0a9ac4062/tests/test_plugins.py#L7
